Notebooks/FoFCatalogMatching Histogram.ipynb

```diff
@@ -44,7 +43,7 @@
     }
    ],
    "source": [
-    "inputCatalog = input_catalog_reader.InputCatalogReader()\n",
+    "inputCatalog = GCRCatalogs.load_catalog('dc2_reference_run1.2p')\n",
```

Comment:
I think it's worth renaming the variables in this Notebook to refer to "referenceCatalog" and "...ReferenceCat" instead of "inputCatalog" and "...InputCat" to match the overall naming we're aiming for.